src/main/java/ch/njol/skript/SkriptCommand.java

```diff
@@ -146,6 +147,17 @@ public boolean onCommand(CommandSender sender, Command command, String label, St
 						});
 				}
 
+				else if (args[1].equalsIgnoreCase("changed")) {
+					reloading(sender, "scripts that have changed since last load");
+
+					ScriptLoader.reloadChangedScripts(ScriptLoader.getLoadedScripts(), OpenCloseable.combine(logHandler, timingLogHandler))
+								.thenAccept(info -> {
+									if (info.files == 0)
+										Skript.warning(Skript.m_no_scripts.toString());
```

Comment:
We'll have to use a different message here (could be at something like `skript command.reload.no changes since last load`)

Message would probably be something like `No script changes have occurred since the last reload`. This would be printed instead of the `reloaded()` method call (switch this to an if/else)